<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="Mamun79s-MBP.lan1" name="com.walgreens.testLayer.LoginPageTest" tests="2" failures="1" timestamp="10 Mar 2019 22:18:55 GMT" time="8.266" errors="0">
  <testcase name="test" time="8.266" classname="com.walgreens.testLayer.LoginPageTest">
    <failure type="java.lang.AssertionError" message="
Expected: a string starting with &quot;Walgreens.com | Walgreens&quot;
     but: was &quot;Sign In or Register to Get Started Using Walgreens.com | Walgreens&quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string starting with "Walgreens.com | Walgreens"
     but: was "Sign In or Register to Get Started Using Walgreens.com | Walgreens"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
at com.walgreens.controller.LoginPageController.verifyMyAccountPageTitle(LoginPageController.java:57)
at com.walgreens.testLayer.LoginPageTest.test(LoginPageTest.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test2" classname="com.walgreens.testLayer.LoginPageTest">
    <skipped/>
  </testcase> <!-- test2 -->
</testsuite> <!-- com.walgreens.testLayer.LoginPageTest -->
